gh-103636: issue warning for deprecated calendar constants (#103833)
python/cpython: Doc/whatsnew/3.12.rst

  yielding tasks.
  (Contributed by Kumar Aditya in :gh:`78530`.)

calendar
--------

> * Add enums for month and day. For example: ``JANUARY`` and ``MONDAY``.

Reviewer: This looks like global variables, not clear they are enum members.
What about making links and letting people go see the values:

```suggestion
* Add enums :data:`~calendar.Month` and :data:`~calendar.Day`.
```

Author: sure

  exception instance, rather than to a ``(typ, exc, tb)`` tuple.
  (Contributed by Irit Katriel in :gh:`103176`.)

> calendar

Reviewer 2: Let's keep the module list in alphabetical order, so move this section up.

  exception instance, rather than to a ``(typ, exc, tb)`` tuple.
  (Contributed by Irit Katriel in :gh:`103176`.)

calendar
--------

> * Added Enum for Month and Day

Reviewer 2: ```suggestion
* Add enums for month and day. For example: ``JANUARY`` and ``MONDAY``.
```

  Python 3.14, when ``'data'`` filter will become the default.
  See :ref:`tarfile-extraction-filter` for details.

* ``January`` and ``February`` is deprated from :mod:`calendar`

>   use ``JANUARY`` and ``FEBRUARY`` instead

Reviewer: deprecated* 

Please add some commas and periods 🙂

  Python 3.14, when ``'data'`` filter will become the default.
  See :ref:`tarfile-extraction-filter` for details.

> * ``January`` and ``February`` is deprecated from :mod:`calendar`,
>   use ``JANUARY`` and ``FEBRUARY`` instead.

Reviewer 2: ```suggestion
* ``January`` and ``February`` are deprecated from :mod:`calendar`,
  use ``JANUARY`` and ``FEBRUARY`` instead.
  (Contributed by Prince Roshan in :gh:`103636`.)
```

  Python 3.14, when ``'data'`` filter will become the default.
  See :ref:`tarfile-extraction-filter` for details.

> * ``January`` and ``February`` are deprecated from :mod:`calendar`,
>   use ``JANUARY`` and ``FEBRUARY`` instead.

Reviewer: ```suggestion
* ``calendar.January`` and ``calendar.February`` constants are deprecated and
  replaced by :data:`calendar.Month.JANUARY` and :data:`calendar.Month.February`.
```
